backend/src/components/permissionHelpers.js

```diff
@@ -69,7 +64,8 @@ const permissionHelpers = {
         return `Acronym '${acronymData.acronym}' is not permitted special access to NROS DMS`;
       }
     } catch (error) {
-      log.error(`Error occurred determening permission for WebADE POST. Error: ${error}`);
+      console.log(error);
```

Comment:
console.log

Reply:
Good catch. Also lame that eslint no longer warns on this... fixed.